pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -130,14 +130,44 @@ class SELinuxEventDetails extends React.Component {
                     </div>
                 );
             }
+
+            // Formatted solution
+            let doElement = "";
+
+            // One line usually means one command
+            if (itm.doText && itm.doText.indexOf("\n") < 0)
+                doElement = <pre>{itm.doText}</pre>;
+
+            // There can be text with commands. Command always starts on a new line with '#'
+            // Group subsequent commands into one `<pre>` element.
+            if (itm.doText.indexOf("\n") >= 0) {
```

Comment:
If you are concerned about doText being null, as in the previous `if`, then that will crash. Probably best to check false-ness first and set doElement to null, then do an if for the indexOf, and handle multi-lines in the else.